op-monitorism/faultproof_withdrawals/runbooks/RUNBOOK.md

```diff
@@ -0,0 +1,162 @@
+
+| **Network** | **Severity Level** | **Impact**                                          | **Cause**                             | **Actions**                                                  |
+|-------------|--------------------|-----------------------------------------------------|---------------------------------------|--------------------------------------------------------------|
+| Mainnet     | SEV2               | - Security max reaction time is reduced<br>- May not be able to react properly to an attack | Daemon is not processing withdrawals  | - Understand the issue with the daemon<br>- If necessary, restart the service |
+| Sepolia     | SEV3               | - Security max reaction time is reduced<br>- May not be able to react properly to an attack | Daemon is not processing withdrawals  | - Understand the issue with the daemon<br>- If necessary, restart the service |
+
+##### Alert Description
+This alert monitors the number of withdrawal events that are considered normal in a chain. If the number of withdrawal events goes below a specified threshold, we trigger this alert.
+
+#### Investigation
+The investigation has two parts:
+1. Investigation to triage if the event is a true positive.
+2. Investigation to act.
+
+TODO: Detail the following steps.
+
+### faultproof-withdrawal-forgery-detection-error-unhandled
+
+| **Network** | **Severity Level** | **Impact**                                          | **Cause**                               | **Actions**                                                                                     |
+|-------------|--------------------|-----------------------------------------------------|-----------------------------------------|-------------------------------------------------------------------------------------------------|
+| Mainnet     | SEV2               | - Security max reaction time is reduced<br>- May not be able to react properly to an attack | Too many errors; something may be wrong | - Find out what the error is<br>- Decide if the daemon needs to be patched or configuration changed<br>- If necessary, restart the service |
+| Sepolia     | SEV3               | - Security max reaction time is reduced<br>- May not be able to react properly to an attack | Too many errors; something may be wrong | - Find out what the error is<br>- Decide if the daemon needs to be patched or configuration changed<br>- If necessary, restart the service |
+
+##### Alert Description
+This alert will be triggered when the number of connection errors goes above a specified threshold. Errors should always be very limited or absent in the monitoring. When present, it often means there is an issue with communication between the monitor and the trusted nodes used for monitoring.
+
+#### Investigation
+
+The investigation has two parts:
+1. Investigation to triage if the event is a true positive.
+2. Investigation to act.
+
+TODO: Detail the following steps.
```

Comment:
There is still a TODO here, probably need to be fixed.

Reply:
Removed the sections that are incomplete. Will be added in a second moment